wheel/python/chia_rs/chia_rs.pyi

```diff
@@ -318,17 +318,17 @@ class PrivateKey:
 
 @final
 class SpendConditions:
-    coin_id: bytes
-    parent_id: bytes
-    puzzle_hash: bytes
+    coin_id: bytes32
+    parent_id: bytes32
+    puzzle_hash: bytes32
     coin_amount: int
     height_relative: Optional[int]
     seconds_relative: Optional[int]
     before_height_relative: Optional[int]
     before_seconds_relative: Optional[int]
     birth_height: Optional[int]
     birth_seconds: Optional[int]
-    create_coin: list[tuple[bytes, int, Optional[bytes]]]
+    create_coin: list[tuple[bytes32, int, Optional[bytes]]]
     agg_sig_me: list[tuple[G1Element, bytes]]
     agg_sig_parent: list[tuple[G1Element, bytes]]
     agg_sig_puzzle: list[tuple[G1Element, bytes]]
```

Comment:
Make lines 344-346 use bytes32

Reply:
I can't suggest changes because I'm on my phone

Reply:
the constructor is deliberately made to take `bytes` for initializing the object. It's implemented in `def transform()` in `generate_type_stubs.py`. Since the conversion from python -> rust already validates the length of 32, requiring the python side to also first cast to `bytes32` would be unnecessary

Reply:
i see, dismissing.